### ISSUE TITLE
Epic Failure

### ISSUE THREAD

**Issue by bondy2108** [Alexqw/xbmc-gamepass#198]
Hey Alex,
Im running Kodi 15.1 and have your latest GamePass installed (running on Amazon FireTv)
Everytime I try to run the app it comes up with Epic Failure message.
Any ideas? Cheers bud

**Comment by Alexqw** [comment]
@bondy2108 When you say latest, do you meant latest from git?

To take a look at what's going on, please enable logging in Kodi and in the addon. Then provide a [log file](http://kodi.wiki/view/Log_file).

You also left a comment in another issue that was specific to Game Pass Domestic users. Are you using Game Pass Domestic?

---Alex

**Comment by bondy2108** [comment]
Hi mate,
Yep the latest from git... 
How do I get a log file from Kodi on the fire stick and send it you? 
 
 
 
 
-------Original Message-------
 
From: Alex Waite
Date: 08/26/15 16:32:44
To: Alexqw/xbmc-gamepass
Cc: bondy2108
Subject: Re: [xbmc-gamepass] Epic Failure (#198)
 
@bondy2108 When you say latest, do you meant latest from git?
To take a look at what's going on, please enable logging in Kodi and in the
addon. Then provide a log file.
You also left a comment in another issue that was specific to Game Pass
Domestic users. Are you using Game Pass Domestic?
---Alex

**Comment by Alexqw** [comment]
The quote myself from before (which includes a link)

> [P]lease enable logging in Kodi and in the addon. Then provide a [log file](http://kodi.wiki/view/Log_file).

---Alex

**Comment by bondy2108** [comment]
Mate,
Unbelievably after I turned on Kodi log program, I launched your app and its
working! 
Thank god for that haha you just saved my spring! (I'm in Australia)
Cheers bud 
 
 
 
 
-------Original Message-------
 
From: Alex Waite
Date: 08/26/15 22:03:43
To: Alexqw/xbmc-gamepass
Cc: bondy2108
Subject: Re: [xbmc-gamepass] Epic Failure (#198)
 
The quote myself from before (which includes a link)
[P]lease enable logging in Kodi and in the addon. Then provide a log file.
---Alex
